README.md

```diff
@@ -61,65 +21,56 @@ chmod +x install
 ./install
 ```
 
-# :beginner: Getting Started
+# Getting Started
 
 Oh My Fish includes a small utility `omf` to fetch and install new packages and themes.
 
-## `omf update`
+### `omf update`
```

Comment:
Have another idea? I often see people using list, but open to other solutions. :D